generate_dip.py

```diff
@@ -103,15 +106,25 @@ def _uuid(identifier: str) -> str:
 
         uuid_pkg = _uuid('pkg')
         uuid_pads = [_uuid('pad-{}'.format(p)) for p in range(1, pin_count + 1)]
-
+        
+        # IPC-7251 name  Example: DIP762W52P254L1905H508Q14B
+        # DIP + width + W lead_width + P spacing + L pin_package_offset*2 +(pin_count/2)* spacing + H component_height + Q pin_count B
+        DIP =  float(width) *100
+        W   =  lead_width
+        P   =  spacing*100
+        L   =  (pin_count/2) * P + pin_package_offset *200
+        H   =  component_height
+        Q   =  pin_count
+
+        
         # General info
         lines.append('(librepcb_package {}'.format(uuid_pkg))
-        lines.append(' (name "{}-{} {}mm")'.format(name, pin_count, width))
+        lines.append(' (name "DIP{:.0f}W{:.0f}P{:.0f}L{:.0f}H{:.0f}Q{}B")'.format(DIP,W,P,L,H,Q))
```

Comment:
Is the trailing `B` there for the density level? I'd drop that, since a package can have multiple density levels as footprint variants.